src/main/cpp/src/hive_hash.cu

```diff
@@ -238,7 +268,9 @@ class hive_device_row_hasher {
 
       __device__ int get_idx_to_process() { return _idx_to_process; }
 
-      __device__ cudf::column_device_view get_column() { return _column; }
+      __device__ cudf::size_type get_col_idx() { return _col_idx; }
+
+      __device__ cudf::size_type get_row_idx() { return _row_idx; }
```

Comment:
These two functions and `get_hash` need to be `const`.

Reply:
This is not a big deal but can trigger warning turning in to compile error in the future.